File: codegen/codegen-elixir/src/test/resources/dsl/client_dispatch_get_name_sigv4.expected.ex
req = HttpServiceRestJson1.encode_get_name_request(input)
signed_req = sign_request(config, :get_name, req)

case RuntimeHttp.dispatch(config, signed_req) do
  {:ok, resp} -> HttpServiceRestJson1.decode_get_name_response(resp)
  {:error, reason} -> {:error, reason}
end

File: codegen/codegen-erlang/src/test/resources/dsl/client_dispatch_get_name_sigv4.expected.erl
Req = http_service_rest_json_1:encode_get_name_request(Input),
SignedReq = sign_request(Config, get_name, Req),
case runtime_http:dispatch(Config, SignedReq) of
    {ok, Resp} ->
        http_service_rest_json_1:decode_get_name_response(Resp);
    {error, Reason} ->
        {error, Reason}
end
